jenkins/jobs/dev_env_integration_tests.pipeline

```diff
@@ -24,11 +24,9 @@ pipeline {
   agent { label agent_label }
   environment {
     METAL3_CI_USER="metal3ci"
-    REPO_ORG = "${PROJECT_REPO_ORG}"
-    REPO_NAME = "${PROJECT_REPO_NAME}"
-    UPDATED_REPO = "${ghprbAuthorRepoGitUrl}"
-    REPO_BRANCH = "${ghprbTargetBranch}"
-    UPDATED_BRANCH = "${ghprbActualCommit}"
+    REPO_ORG = "${env.REPO_OWNER}"
+    REPO_NAME = "${env.REPO_NAME}"
+    REPO_BRANCH = "${env.PULL_BASE_REF}"
```

Comment:
This varibale needed to run for PR tests. Please add it here. 
PR_ID = "${env.PULL_NUMBER}"